### DISCUSSION TITLE
"unknown" Sensors when syncing w/ Teslamate

### DISCUSSION THREAD

**Discussion by sockmonkey0223** [alandtse/tesla#847] [Q&A]
I have the integration syncing with Teslamate/MQTT for 2 cars. I notice that quite often various sensors go to "unknown" status for one or both vehicles. "odometer" is a common one. Whenever I check the source values in MQTT explorer I see that they exist and are retained. Is this normal?

I don't remember this behavior before I used the Teslamate/MQTT sync, but maybe I just never noticed it.

**Comment by kzaoaai** [comment]
i think tesla should check the teslamate/cars/$car_id/since timestamp and decide whether to sync from teslamate or not accordingly. I had a stale teslamate instance and my car device_tracker was jumping between two locations because of that. Finally pinpointed the reason.
